### ISSUE TITLE
[BUG] Unhiding the SSID doesn't work

### ISSUE THREAD

**Issue by danielraffel** [Monitor-My-Solar/monitormysolar#44]
### Affected Component

Dongle

### Version Information

2.1.4/2.0.57

### Media Attachments

![image](https://github.com/user-attachments/assets/06b986c5-4bca-4c9d-af1d-b5d7ec6eb530)

### MQTT Output

```shell
n/a
```

### Detailed Description

I attempted to hide my SSID
The first time I checked 'hide' SSID checkbox and pressed save I saw an error that it failed.
The second time I checked 'hide' SSID checkbox and pressed save it worked and the device rebooted and SSID was hidden
When I attempted to unhide the SSID by deselecting the SSID checkbox and pressing save I got an error "Failed to save WiFi settings: Failed to save settings"

tldr I cannot show the SSID now 

### Issues

- [X] I agree i have checked the Issues Tab to see if this already Exists

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by zakery292** [comment]
Fixed in 2.5.5 > dongle FW
